crates/api_models/src/admin.rs

```diff
@@ -342,28 +338,26 @@ pub struct PaymentMethods {
     pub payment_experience: Option<Vec<api_enums::PaymentExperience>>,
 }
 
-/// List of enabled and disabled currencies
+/// List of enabled and disabled currencies, empty in case all currencies are enabled
```

Comment:
this could be an enum with serde tag as type with 3 variants, `EnableAll`, `EnableOnly(Vec<_>)`, `DisableOnly(Vec<_>)`.

Reply:
can be with only `EnableOnly(Vec<_>)` and `DisableOnly(Vec<_>)`  as enum field
not passing means `enable_all`

Reply:
@jarnura @bernard-eugine  ?

Reply:
`enable_all:true` is default, and they can pass `enable_all:false` means  disable all.

Reply:
@bernard-eugine Can you confirm once do we need disable all?

Reply:
DIsable all is not required, verified with @bernard-eugine